src/components/uxiconbaritem/uxiconbaritem.hbs

```diff
@@ -0,0 +1,4 @@
+<a class="item">
+    <img src="{{src}}">
+    <label>{{label}}</label>
```

Comment:
Was to restrict input, but what's nicer?
On 19 Mar 2015 18:24, "nathf" <notifications@github.com> wrote:

> In src/components/uxiconbaritem/uxiconbaritem.hbs
> <https://github.com/themacclesoft/ractive-foundation/pull/34#discussion_r26733717>
> :
>
> > @@ -0,0 +1,4 @@
> > +<a class="item">
> > +    <img src="{{src}}">
> > +    <label>{{label}}</label>
>
> Are you enforcing passing a label to control input? Rather than just
> passing it through {{yield}}?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/themacclesoft/ractive-foundation/pull/34/files#r26733717>
> .
>